src/components/LeftSidebar/LeftSidebar.vue

```diff
@@ -470,8 +473,9 @@ export default {
 					params: { token: conversation.token },
 				}).catch(err => console.debug(`Error while pushing the new conversation's route: ${err}`))
 			} else {
-				// For other types we start the conversation creation dialog
-				EventBus.$emit('new-group-conversation-dialog', item)
+				// For other types, show the modal directly
+				this.$refs.newGroupConversation.showModalForItem(item)
+				this.toggleNewGroupConversation(true)
```

Comment:
```suggestion
        this.$refs.newGroupConversation.showModalForItem(item)
```

this method already opens modal by this line, so we don't need to do it twice:
```js
	this.$emit('open-modal')
```